### PR TITLE
Fixes #24022: File content directive add 4 spaces at the beginning of the line 

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/24022

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

File: techniques/fileDistribution/fileTemplate/1.0/fileTemplate.ps1.st [hunk 1]
```diff
@@ -26,8 +26,8 @@ function fileTemplate_&RudderUniqueID& {
     "&isRaw&" };separator=","& )
 
     $rawSources = @(
-    &FILE_TEMPLATE_RAW_TEMPLATE:{rawSource |
-    "&rawSource&" };separator=","& )
+ &FILE_TEMPLATE_RAW_TEMPLATE:{rawSource |
```
Comment (ncharles) [inline techniques/fileDistribution/fileTemplate/1.0/fileTemplate.ps1.st:29]:
```suggestion
&FILE_TEMPLATE_RAW_TEMPLATE:{rawSource |
```

**Review by ncharles — CHANGES_REQUESTED** [review]
you need exactly 0 spaces

**Comment by ElaadF** [comment]
Commit modified

**Review by ncharles — APPROVED** [review]

**Pull request merged**